### PR TITLE
[4.0] Correcting accessibility in sidebar menu

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/27045#issuecomment-554322139

### Summary of Changes
Using specific lang strings when adding module and menu item

### Testing Instructions
Look at source around sr-only for the modules and menus wether using alternate or default preset in admin menu

### CODE REVIEW

File: administrator/language/en-GB/en-GB.mod_menu.ini [hunk 1]
```diff
@@ -21,7 +21,8 @@ MOD_MENU_COM_CONTENT_MANAGE="Content Settings"
 MOD_MENU_COM_CONTENT_MISC="Modules & Media"
 MOD_MENU_COM_CONTENT_NEW_ARTICLE="Add New Article"
 MOD_MENU_COM_CONTENT_NEW_CATEGORY="Add New Category"
-MOD_MENU_COM_CONTENT_NEW_MODULE="Add New Module"
+MOD_MENU_COM_CONTENT_NEW_ADMIN_MODULE="Add New Administrator Module"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.mod_menu.ini]: Remove `New`. See [comment](https://github.com/joomla/joomla-cms/pull/25865#issuecomment-522782938).

Comment (infograf768) [reply administrator/language/en-GB/en-GB.mod_menu.ini]: Can do, but it means all the strings there for new items should also be modified.
Screen Shot 2019-11-16 at 07 45 17

Shall we do that in this PR?
@brianteeman ??

Comment (Quy) [reply administrator/language/en-GB/en-GB.mod_menu.ini]: @infograf768 Either way will work.

Comment (infograf768) [reply administrator/language/en-GB/en-GB.mod_menu.ini]: Will modify all tomorrow

**Comment by infograf768** [comment]
@Quy @brianteeman 
Good to test now.

File: administrator/language/en-GB/en-GB.mod_menu.ini [hunk 2]
```diff
@@ -134,8 +135,8 @@ MOD_MENU_MANAGE_REDIRECTS="Redirects"
 MOD_MENU_MASS_MAIL_USERS="Mass Mail Users"
 MOD_MENU_MEDIA_MANAGER="Media"
 MOD_MENU_MENU_MANAGER="Manage"
-MOD_MENU_MENU_MANAGER_NEW_MENU="Add New Menu"
-MOD_MENU_MENU_MANAGER_NEW_MENU_ITEM="Add New Menu Item"
+MOD_MENU_MENU_MANAGER_NEW_ADMIN_MENU_ITEM="Add New Administrator Menu Item"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_menu.ini]: do you not want to remove the word "new" here and on the next string as well?

Comment (infograf768) [reply administrator/language/en-GB/en-GB.mod_menu.ini]: oops

Comment (infograf768) [reply administrator/language/en-GB/en-GB.mod_menu.ini]: Done

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on daccdf6b419148d29a7b0a4ce235013ffc44724c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27079">issues.joomla.org/tracker/joomla-cms/27079</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on daccdf6b419148d29a7b0a4ce235013ffc44724c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27079">issues.joomla.org/tracker/joomla-cms/27079</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27079">issues.joomla.org/tracker/joomla-cms/27079</a>.</sub>

File: administrator/language/en-GB/en-GB.mod_menu.ini [hunk 3]
```diff
@@ -36,10 +37,10 @@ MOD_MENU_COM_PRIVACY_REQUESTS="Requests"
 MOD_MENU_COM_TEMPLATES_SUBMENU_STYLES="Styles"
 MOD_MENU_COM_TEMPLATES_SUBMENU_TEMPLATES="Templates"
 MOD_MENU_COM_USERS="Users"
-MOD_MENU_COM_USERS_ADD_GROUP="Add New Group"
-MOD_MENU_COM_USERS_ADD_LEVEL="Add New Access Level"
+MOD_MENU_COM_USERS_ADD_GROUP="Add Group"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.mod_menu.ini:40]: Please confirm that these 2 strings are not used.

Comment (infograf768) [reply administrator/language/en-GB/en-GB.mod_menu.ini:40]: They are not used for the moment, but they could if we add some quicktasks in the presets.
To be discussed in another Issue/PR imho.

**Pull request merged**